data-prepper-main/src/main/java/org/opensearch/dataprepper/DataPrepperExecute.java

```diff
@@ -19,7 +21,14 @@ public class DataPrepperExecute {
     public static void main(final String ... args) {
```

Comment:
This might be a confusing experience, if users pass in the arguments while running Data Prepper they'll be ignored. Maybe have a warning message that arguments are ignored.

Reply:
Good point. I will address this in a coming PR for a closely related issue #1736. 
I'm also thinking about accepting command line arguments again as I noticed it might benefits certain use cases where config files are swapped out frequently (e.g. during development).